governance/votes/Executive vote - March 4, 2022.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Template - [Executive Vote] Core Unit Budget Distributions, Recognized Delegate Compensation, MOMC Parameter Changes - March 4, 2022
+summary: Core Unit Budget Distributions, Recognized Delegate Compensation Distribution for February, MOMC Parameter Changes for WSTETH-A and DIRECT-AAVEV2-DAI
+date: $2022-03-04T00:00:00.000Z
+address: "$spell_address"
+---
+
+# [Executive Proposal] Core Unit Budget Distributions, Recognized Delegate Compensation, MOMC Parameter Changes - March 4, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+## Executive Summary
+
+If this executive proposal passes, the following **outcomes** will be executed:
+
+- A total of 397,184 Dai (itemized below) will be transferred to multiple Core Units.
+- A total of 5,747,562 Dai (itemized below) will be streamed to multiple Core Units.
+- The existing DAI budget stream for the Risk Core Unit will be revoked.
+- A total of 57,665 DAI (itemized below) will be transferred to eight Recognized Delegates as part of the [Delegate Compensation Trial](https://forum.makerdao.com/t/signal-request-extend-the-delegate-compensation-trial/12686).
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+- Parameter changes (detailed below) from the latest proposal from the MakerDAO Open Market Committee, including [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) increases, [Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) increases, and a [Target Borrow Rate](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar) decrease.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+---
+
+## Proposal Details
```

Comment:
Is it worth bolding or etherscan-linking the listed addresses here? Probably fine as is, but the alphanumeric string can be a little nauseating in plain text